autogen/agentchat/contrib/capabilities/context_handling.py

```diff
@@ -46,7 +46,7 @@ def add_to_agent(self, agent: ConversableAgent):
         """
         Adds TransformChatHistory capability to the given agent.
         """
-        agent.register_hook(hookable_method="process_all_messages", hook=self._transform_messages)
+        agent.register_hook(hookable_method="process_all_received_messages", hook=self._transform_messages)
```

Comment:
Isn't it the case that long-context handling needs to process both sent and received messages?

Reply:
Thanks. Renamed.